promql/promqltest/testdata/functions.test

```diff
@@ -759,32 +752,32 @@ eval instant at 1m sum_over_time(metric9[1m])/count_over_time(metric9[1m])
   {} -Inf
 
 eval instant at 1m avg_over_time(metric10[1m])
-  {} 0
+  {} 9.98846567431158e+307
 
 eval instant at 1m sum_over_time(metric10[1m])/count_over_time(metric10[1m])
-  {} 0
+  {} 9.98846567431158e+307
 
 # Test if very big intermediate values cause loss of detail.
```

Comment:
So yeah, in these tests, too, we want the same samples in the range, so the range duration has to be extended (rather than adjusting the result).

Reply:
And again, the purpose of this test is to cover all four values, so the test expression should be something like `eval instant at 45s sum_over_time(metric[1m])`, and the result should be unchanged.

Reply:
Resolved. I added such cases:
```
eval instant at 45s avg_over_time(metric10[1m])
  {} 0

eval instant at 1m avg_over_time(metric10[2m])
  {} 0

eval instant at 45s sum_over_time(metric10[1m])/count_over_time(metric10[1m])
  {} 0

eval instant at 1m sum_over_time(metric10[2m])/count_over_time(metric10[2m])
  {} 0
```
Both `metric10[1m]` at 45s and `metric10[2m]` at 1m shall evaluate 2 samples, to make them `0/2`.